Sources/SwiftCLI/ParameterFiller.swift

```diff
@@ -62,11 +62,14 @@ public class DefaultParameterFiller: ParameterFiller {
     }
     
     func wrongArgCount(signature: CommandSignature, got: Int) -> CLI.Error {
-        let requiredCount = signature.required.count
+        var requiredCount = signature.required.count
+        if signature.collected?.required == true {
+            requiredCount += 1
+        }
         let optionalCount = signature.optional.count
         
         let plural = requiredCount == 1 ? "argument" : "arguments"
-        if signature.collected != nil {
+        if let collected = signature.collected {
```

Comment:
Just noticed this is generating a warning regarding unused value `collected`.

This change is no longer required after 7f106725e8c700e8eb269840c7a9a9a3df540ece, I can do another PR later.

Reply:
Fixed in 91c9a0b! Now tagged as 4.0.4

Reply:
Cheers, and thanks for the quick release 😀